Sampler/Templates/Sampler/CHANGELOG.md.template

```diff
@@ -1,26 +1,30 @@
-# Changelog
+# Changelog for {ModuleName}
 
-All notable changes to this project will be documented in this file.
-
-The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+The format is based on and uses the types of changes according to [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
 ### Added
+
```

Comment:
I don't think this is
- 1. valid keepachangelog
- 2. valid md

I think titles need a newline after, but I may be wrong.

Reply:
It should be a valid keepachangelog, and the sections should be okay.
Real world one here: https://raw.githubusercontent.com/dsccommunity/SqlServerDsc/master/CHANGELOG.md